copier.go

```diff
@@ -178,8 +181,64 @@ func set(to, from reflect.Value) bool {
 		} else if from.Kind() == reflect.Ptr {
 			return set(to, from.Elem())
 		} else {
-			return false
+			cft := cFromTo{
+				ToType:   to.Type(),
+				FromType: from.Type(),
+			}
+
+			cFunc := copiersFunc[cft]
+
```

Comment:
How about process this at the beginning of is `set` method for scanner.Scan may be failed?

Reply:
Agreed, scanner.Scan failures on a data model I can't change are exactly why I would prefer this to come up front.